src/query/sql/src/executor/format.rs

```diff
@@ -662,3 +666,18 @@ fn unnest_to_format_tree(
         children,
     ))
 }
+
+fn runtime_filter_source_to_format_tree(
+    plan: &RuntimeFilterSource,
+    metadata: &MetadataRef,
+    prof_span_set: &ProfSpanSetRef,
+) -> Result<FormatTreeNode<String>> {
+    let children = vec![
+        to_format_tree(&plan.left_side, metadata, prof_span_set)?,
+        to_format_tree(&plan.right_side, metadata, prof_span_set)?,
+    ];
+    Ok(FormatTreeNode::with_children(
+        "RuntimeFilterSource".to_string(),
+        children,
+    ))
+}
```

Comment:
just curious, for other databases support runtime filter, will this node show up in their `explain <query>`?

Reply:
Yes